examples/I/README.txt

```diff
@@ -0,0 +1,36 @@
+Allocation tiering based on memory-characteristics.
+
+Starting from Linux Kernel version 5.5, the NUMA nodes concept was enhanced with
+new memory attributes: latency, capacity and bandwidth. We could use them to
+allocate data used in different scenarios on different types of memory. Note, that here
```

Comment:
Note, that ~~*here*~~ we don't explicitly say which kind of medium (e.g. DRAM or PMEM) we want to use **here**